OpenTabletDriver.Web/Views/Wiki/FAQ/Windows.cshtml

```diff
@@ -76,24 +118,32 @@
 <hr/>
 
 <h3 class="wiki-nav-item pb-2" id="connecting-to-daemon">
-    Stuck at connecting to Daemon
+    It says it fails to connect to the daemon?
 </h3>
 <div class="ms-3">
     <p>
-        This is usually caused by one of two things. Either you didn't follow the <a href="/Wiki/Install/Windows">installation guide</a> correctly
-        or the settings file is corrupted. To remedy this case follow this below.
+        This can be caused by a few main problems. Primarily due to incorrectly following the <a href="/Wiki/Install/Windows">installation guide</a>.<br/><br/>
```

Comment:
```suggestion
        This can be caused by a few different problems. Primarily, this will result from incorrectly following the <a href="/Wiki/Install/Windows">installation guide</a> on installation.<br/><br/>
```